src/Microsoft.ML.StandardLearners/Standard/LogisticRegression/LogisticRegression.cs

```diff
@@ -397,4 +426,113 @@ public static CommonOutputs.BinaryClassificationOutput TrainBinary(IHostEnvironm
                 () => LearnerEntryPointsUtils.FindColumn(host, input.TrainingData.Schema, input.WeightColumn));
         }
     }
+
+    /// <summary>
+    /// Computes the standart deviation matrix of each of the non-zero training weights, needed to calculate further the standart deviation,
```

Comment:
>standart deviation [](start = 21, length = 18)

`standard deviation` ?

Reply:
Here and in a few places.

---
In reply to: [232703545](https://github.com/dotnet/machinelearning/pull/1392#discussion_r232703545) [](ancestors = 232703545)